src/components/Router.js

```diff
@@ -88,7 +88,7 @@ export default class Router extends Component {
         div(['Welcome to the new FireCloud interface, powered by Terra. All of your workspaces are available. ',
           link({
             ...Utils.newTabLinkProps,
-            href: 'https://broadinstitute.zendesk.com/hc/en-us/sections/360003528231-FireCloud-users-Find-out-what-s-new-in-Terra',
+            href: 'https://support.terra.bio/hc/en-us/sections/360003528231-FireCloud-users-Find-out-what-s-new-in-Terra',
```

Comment:
```suggestion
            href: 'https://support.terra.bio/hc/en-us/sections/360004482892-Terra-for-FireCloud-Users',
```

Reply:
Is it supposed to be this? https://support.terra.bio/hc/en-us/articles/360022506932-Uniquely-Terra-What-s-new-and-coming-soon

Reply:
This is what Comms recommended, I could go either way. Maybe @bradtaylor could weigh in?

Reply:
Brad wants the overall page rather than the specific article! (Kate's link)